src/style.css

```diff
@@ -1,3 +1,75 @@
 body {
   margin: 0;
 }
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-sizing: border-box;
```

Comment:
move to `*` selector

Reply:
It doesn't work in my code in such way. All my elements are located without any space between them and without any margins at all